### PR TITLE
Timestamp and duration

### PR DESCRIPTION
**Pull request by spenced**

### CODE REVIEW

**Review by crwr45 — APPROVED** [review]

**Comment by spenced** [comment]
Requesting re-review after squish and force push.

File: src/testdrive/run.py [hunk 1]
```diff
@@ -132,15 +128,13 @@ def main():
...
-            if not args.timeless:
-                start = timenow()
+            start = timenow()
             result = drive(testimpl, *test_args)
-            if not args.timeless:
-                end = timenow()
+            end = timenow()
             result['id'] = id_
-            if not args.timeless:
+            if 'timestamp' not in result:
```
Comment (jnunyez) [inline src/testdrive/run.py:135]: Doubt: This case `'timestamp' not in result` only would apply now for for environment checks right?

**Review by jnunyez — APPROVED** [review]
Asked a dout. LGTM.

File: src/testdrive/run.py (hunk 1 above)
Comment (spenced) [reply src/testdrive/run.py:135]: Maybe, we would have to check how they are implemented. In any case the logic now is "if a timestamp wasn't given by the test, we must add one".

**Pull request merged**